packages/utils/src/__tests__/object.test.js

```diff
@@ -359,4 +360,19 @@ describe('object', () => {
       });
     });
   });
+
+  describe('filterValues', () => {
+    const object = { a: 1, b: 2, alpha: 1 };
+
+    const testData = [
+      ['no entry passes', value => value === 3, {}],
```

Comment:
Could split this into two tests: `no entries in object`, and `no entry passes`

Reply:
Good idea!